scripts/metric_reporter/main.py

```diff
@@ -84,5 +132,13 @@ def main(config_file: str = "config.ini") -> None:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Run the Metric Reporter")
     parser.add_argument("--config", help="Path to the config.ini file", default="config.ini")
+    parser.add_argument("--output_csv", help="Output CSV files", type=bool, default=None)
+    parser.add_argument(
+        "--update_bigquery", help="Update BigQuery tables", type=bool, default=None
+    )
```

Comment:
Or a possibly better regex (since I still had the tab open)&hellip; but again, its a terrible regex because it is (and I am) very naively grepping and they might not be comparable flags.

```sh
git grep -iEho "\-{2}[a-z_-]{1,}" | grep -vE "^[-]+$" | grep "[a-z][-_][a-z]" | sort | uniq

--config-file
--cov-report
--no-root
--output_csv
--report-type
--update_bigquery
```

Reply:
Hyphens it is! see https://github.com/mozilla/ecosystem-test-scripts/pull/37/commits/de6a04f399327d69d0edfd6ce001fb82d4d6b969